admin/services/sas-token.service.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const azure = require('azure-storage')
+const moment = require('moment')
+const config = require('../config')
+
+const addPermissions = azure.QueueUtilities.SharedAccessPermissions.ADD
+const storageConnection = process.env.AZURE_STORAGE_CONNECTION_STRING
```

Comment:
It would be nice to get a default in the config file for this, otherwise you wouldn't know what env variable to set.

Reply:
Jon has changed this so it is no longer required, for the benefit of unit testing.
As we progress with the overall implementation we can re-assess this.

Reply:
Good spot, I'll fix this on the next PR.